config/routes.rb

```diff
@@ -12,5 +12,6 @@
   resources :categories, only: [:show]
 
   resources :subscribers, only: [:new, :create]
+  resources :users, only: [:index, :update]
```

Comment:
This should be in the `content_manegment` namespace

Reply:
updated